### DISCUSSION TITLE
How to use this?

### DISCUSSION THREAD

**Discussion by EurikaDude** [Ziyph/RemoveDuplicateFrames#10] [General]
I have FFmpeg and RemoveDuplicateFrames in one folder but i don't know what i'm supposed to do next. I can't open them either, but i don't know anything about programming or scripts. Can someone help?

**Comment by Ziyph** [comment]
Hi, I hope you're doing well.

Now that you have the FFmpeg and the script in the same folder, the next step is to run the script.

1. Run **Command Prompt** on Windows or **Terminal** on Mac or Linux.

2. Type `cd [drag the script's folder into the window]` and enter.

![image](https://github.com/Ziyph/RemoveDuplicateFrames/assets/57063197/2fb839cb-8924-4942-a592-58c485316365)

3. Type `python3 RemoveDuplicateFrames-v1.1.1.py` and enter.

![image](https://github.com/Ziyph/RemoveDuplicateFrames/assets/57063197/fc9e1061-330e-4838-a765-0ed7648d61d6)

4. Profit!

![image](https://github.com/Ziyph/RemoveDuplicateFrames/assets/57063197/7d197c91-aac1-4491-9b48-d1e065e41b1b)

If you still need help, feel free to ask any questions below. 😄

**Reply by EurikaDude** [reply]
Hi there! Sorry to get back to you so late on this, however when i follow the steps of this my command prompt says "File path not found". Am i missing something perhaps?
![image](https://github.com/Ziyph/RemoveDuplicateFrames/assets/116967768/b7ee0051-5a60-4c48-a95a-d4d7cc16cb7a)
